app/kumactl/pkg/cmd/root_context.go

```diff
@@ -134,6 +134,9 @@ func (rc *RootContext) CurrentDataplaneTokenClient() (tokens.DataplaneTokenClien
 	if err != nil {
 		return nil, err
 	}
+	if !components.Apis.DataplaneToken.Enabled() {
+		return nil, errors.New("Dataplane Token Server in Kuma CP is disabled. Either enable the server or run Kuma DP without token.")
```

Comment:
> or run Kuma DP without token

Why is this message about `Kuma DP` ? It's `app/kumactl/pkg/cmd/root_context.go`

Reply:
When dataplane token server is disabled user can either enable the server to generate token or just run kuma dp without token. Do you think we skip the running kuma dp without token option?